tests/rados/test_config_parameter_chk.py

```diff
@@ -24,10 +26,18 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     osd_list = []
     mclock_profile = ["balanced", "high_recovery_ops", "high_client_ops"]
-    # To get the parameters from the ini
     rados_object = RadosOrchestrator(node=cephadm)
     mon_object = MonConfigMethods(rados_obj=rados_object)
-
+    cmd_get_ceph_ver = "ceph -v"
```

Comment:
This method would require code change added for every release. Can we pass the ini file from the test suite?

Reply:
Passing the ini file from the test suite.